### PR TITLE
Fix the linter

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: smoke-test/metric_service_smoke_test.js [hunk 1]
```diff
@@ -16,23 +16,24 @@
...
+    throw new Error('Usage: GCLOUD_PROJECT=<project_id> node #{$0}');
   }
-  var projectId = process.env.GCLOUD_PROJECT;
+  let projectId = process.env.GCLOUD_PROJECT;
 
   it('successfully makes a call to the service using promises', done => {
     const monitoring = require('../src');
 
-    var client = new monitoring.v3.MetricServiceClient({
+    let client = new monitoring.v3.MetricServiceClient({
```
Comment (kinwa91) [inline smoke-test/metric_service_smoke_test.js]: client is never reassigned within block, use `const`?

File: smoke-test/metric_service_smoke_test.js [hunk 2]
```diff
@@ -16,23 +16,24 @@
...
       // optional auth parameters.
     });
 
     // Iterate over all elements.
-    var formattedName = client.projectPath(projectId);
+    let formattedName = client.projectPath(projectId);
```
Comment (kinwa91) [inline smoke-test/metric_service_smoke_test.js]: Same comment as above^

File: smoke-test/metric_service_smoke_test.js (hunk 1 above)
Comment (JustinBeckwith) [reply smoke-test/metric_service_smoke_test.js]: Right now these are automated changes that will be fixed when we enable `prefer-const` in the .eslintrc.  I'm tired of leaving this feedback too so I'm enabling a lint rule :P

File: smoke-test/metric_service_smoke_test.js (hunk 2 above)
Comment (JustinBeckwith) [reply smoke-test/metric_service_smoke_test.js]: same as above

**Pull request merged**